statistics/table.go

```diff
@@ -505,6 +505,16 @@ func (t *Table) IsOutdated() bool {
 	return false
 }
 
+// ReleaseAndPutToPool releases data structures of Table and put itself back to pool.
+func (t *Table) ReleaseAndPutToPool() {
+	for _, col := range t.Columns {
+		col.FMSketch.DestroyAndPutToPool()
```

Comment:
I think it's safer to set `FMSketch` to `nil` after putting to the pool.

Reply:
Ok to me to improve them in another PR or in another way at your convenience.

Reply:
Done，use ```map.clear``` to clean it.